userguide/one_shot_object_detection/README.md

```diff
@@ -44,81 +45,70 @@ model.save('mymodel.model')
 model.export_coreml('MyCustomOneShotDetector.mlmodel')
 ```
 
-Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
 
+Examples of test image predictions:
 
-```python
-augmented_images = \
-	tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
+![Sample prediction image - 1 and 2](images/sample_prediction_images_1_2.jpg)
 
-augmented_images.explore()
-```
 
-![Explore augmented images](images/augmented_images_explore.jpg)
 
-Some examples of synthetic training data generated:
+## How it works
 
-![Sample augmented images](images/augmented_images_collage.jpg)
+In this section, we will go into detail on what happens under the hood of the `create` method. An OSOD model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.
 
-*Note: In order to view the exact augmented images used to train the model, the same seed should be used which was used to train the model.*
+**Stage 1: Automatic synthetic data generation**
 
+To begin training, a set of synthetic training data needs to be generated using the provided starter image(s). 
+The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. If no background images are provided, a set of default background images are automatically downloaded and used.
 
-To visualize the predictions made on the test set:
+**Stage 2: Train the Object Detector model**
 
+To learn about the training implementation details, refer to the [Object Detector: How it Works](https://apple.github.io/turicreate/docs/userguide/object_detection/how-it-works.html) chapter.
 
-```python
-test_data['image_with_predictions'] = \
-	tc.object_detector.util.draw_bounding_boxes(test_data['image'], test_data['predictions'])
 
-test_data.explore()
-```
 
-![Prediction test data](images/test_data_explore.jpg)
+## Deployment to CoreML
```

Comment:
Why is this in this section. Shouldn't this be in the intro example?

Reply:
Oh good point, I placed it under the intro section.